immunopepper/io_.py

```diff
@@ -10,16 +10,19 @@
 import sys
 import timeit
 
-from .namedtuples import Coord
-from .namedtuples import Filepointer
-
 ### intermediate fix to load pickle files stored under previous version
 import spladder.classes.gene as gene
 import spladder.classes.splicegraph as splicegraph
 import spladder.classes.segmentgraph as segmentgraph
+
+from immunopepper.namedtuples import Coord
+from immunopepper.namedtuples import Filepointer
+
```

Comment:
Technically, the 6 lines of spladder imports and sys.modules additions belong to the same fix, that addresses a data handling issue for inputs coming from different SplAdder versions. It might make sense to keep those together, but otherwise fine to interleave these additional imports here.